### ISSUE TITLE
hwameistor scheduler use 127.0.0.1:6443 to access api-server which is not accessable

### ISSUE THREAD

**Issue by panpan0000** [hwameistor/hwameistor#544]
When it happened, hwameistor scheduler  will stop working.example logs
```
 1 reflector.go:138] k8s.io/client-go/informers/factory.go:134: Failed to watch *v1.Service: failed to list *v1.Service: Get "https://127.0.0.1:6443/api/v1/services?limit=500&resourceVersion=0": dial tcp 127.0.0.1:6443: connect: connection refused
```

because hwameistor-scheduler pod runs in calico network, without access to 127.0.0.1:6443.

it mounts below two host config file as api-server access cert and url information.
```
/etc/kubernetes/admin.conf
/etc/kubernetes/scheduler.conf
```
in those conf file, kubespray initialize it with 127.0.0.1 ip address.

but it's not correct, should use kubernetes service instead .
```
[root@m-master1 ~]# kubectl get svc
NAME            TYPE           CLUSTER-IP       EXTERNAL-IP    PORT(S)          AGE
kubernetes      ClusterIP      100.64.0.1       <none>         443/TCP          5d11h
```

**Comment by panpan0000** [comment]
I know this issue has been fixed in 0.6.1.
but the reference issue is quite un-direct and unclear: https://github.com/hwameistor/hwameistor/issues/521
so I open this one to explain the root cause.

I recommend the hwameistor developer can explain issue root cause and fix more clear.

**Comment by panpan0000** [comment]
But I still confused about the reason why hwameistore sometimes works well , and sometime failed as above.
